src/heartbeat/pulse.js

```diff
@@ -3,10 +3,13 @@ const { Tx, Input, Outpoint, Output } = require('leap-core');
 const getUnspent = require('../api/methods/getUnspent');
 
 module.exports = async (bridgeState, { send }) => {
+  const heartbeatColor = await bridgeState.operatorContract.methods
```

Comment:
pulse is being called by setTimeout many times. indefinitely. No reason to call heartbeatColor() on the contract everytime — it won't change often (if ever)